src/gdcdictionary/schemas/other_clinical_attribute.yaml

```diff
@@ -394,7 +405,6 @@ properties:
     $ref:
       - "_terms.yaml#/dlco_ref_predictive_percent/common"
     type: number
-    maximum: 100
```

Comment:
is there a reason why we are removing the maxes? can percent go over 100?

Reply:
We found out that yesterday from Megan, apparently these are lung capacity tests and they can go over 100